src/analysis_file.py

```diff
@@ -23,12 +25,12 @@ def generate(user, data, csv_by_message_output_path, csv_by_individual_output_pa
                            Metadata(user, Metadata.get_call_location(), time.time()))
 
         # Set the list of keys to be exported and how they are to be handled when folding
+        fold_strategies = OrderedDict()
```

Comment:
Does it matter that they're an ordered dict?

Reply:
I'm guessing not, but you want an explicit ctor call?

Reply:
At this point no, it was just me thinking ahead to the next step. I'll merge this as-is, then add a comment there pointing to this one